website/docs/tutorial/leveling-up.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Leveling up"
```

Comment:
Not sure what Leveling is up for but I want to plug for the courses if it means building on what they learned in the getting started guide?